dev_requirements.txt

```diff
@@ -1,8 +1,10 @@
 setuptools>=45.2.0
 Cython>=0.29.32
 wheel>=0.34.2
-cibuildwheel>=2.11.1
+cibuildwheel>=2.11.2
 Sphinx>=5.0.2
 sphinx-rtd-theme>=1.0.0
 twine>=4.0.1
 bump2version>=1.0.1
+pandas>=1.3.5
+numpy>=1.21.6
```

Comment:
pyarrow and fastparquet dependencies may be missing

Reply:
Done.